### ISSUE TITLE
Reading: Hidden State Variability of Pretrained Language Models Can Guide Computation Reduction for Transfer Learning

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#270]
## 0. Paper
 - paper: [[link](https://aclanthology.org/2022.findings-emnlp.422/)]
 - Findings of EMNLP2023

## 1. What is it?
They propose a metric to predict the optimal (well-specialized) layer for fine-tuning.

## 2. What is amazing compared to previous works?
Their metric does not require any training or hyperparameter tuning.
Their optimal layer effectively performs downstream tasks with 500% more computation cost.

### 3.1 Metric
![スクリーンショット 2023-06-19 13 14 09](https://github.com/a1da4/paper-survey/assets/45454055/969ae34a-68da-42e5-b08e-f10cee38788a)

They define the metric to evaluate the optimal (well-specialized) layer as follows:

 - calculate sequence (sentence) embeddings by averaging all tokens (except for CLS token)
 - clustering by the target labels (blue, red, and yellow in Figure 1)
 - calculate the within-group variability and between-group variability as:
![スクリーンショット 2023-06-19 13 17 06](https://github.com/a1da4/paper-survey/assets/45454055/a8d4a988-0c16-4497-8087-bfd3bf272fd5)

 - based on these scores, they define the task speciality of the layer
![スクリーンショット 2023-06-19 13 18 42](https://github.com/a1da4/paper-survey/assets/45454055/0518ed0f-bbac-4f01-9ca1-62573c71b83e)

### 3.2 Tuning strategy
![スクリーンショット 2023-06-19 13 19 55](https://github.com/a1da4/paper-survey/assets/45454055/b8de47bc-f3df-4f6a-8125-49c856a97889)

Based on the optimal layer, they tried four tuning strategies in Figure 2.

## 4. How did evaluate it?

![スクリーンショット 2023-06-19 13 22 01](https://github.com/a1da4/paper-survey/assets/45454055/e076b610-4964-4411-98af-d330c723eb8d)
Figure 3 shows that their metric is highly correlated with task performance.

![スクリーンショット 2023-06-19 13 24 58](https://github.com/a1da4/paper-survey/assets/45454055/c3bf1cd0-fab6-4362-b575-169e9774e37f)
Figures 4 and 5 show that their tuning strategies (tuning only optimal layers) achieve comparable performance to tuning all of the layers.
